smdebug/tensorflow/utils.py

```diff
@@ -4,18 +4,13 @@
 from enum import Enum
 
 # Third Party
-from tensorflow.contrib.distribute import MirroredStrategy as ContribMirroredStrategy
+import tensorflow as tf
```

Comment:
?
compat.v1?

Reply:
dint add it here to differentiate between tf.compat.v1.mirroredstrat and tf.distributed.mirroredstrat. seemed safe in this file as there were no other tf specific calls (except tf.version and tf.tensor which works with tf 1.x and 2.x).